web-ui/styles/responsive.css

```
@@ -477,6 +508,67 @@ textarea:focus-visible {
+    }
+
+    .session-preview-meta-item {
+        min-width: 0;
+        max-width: 100%;
+        white-space: normal;
+        overflow-wrap: anywhere;
+    }
+
+    .session-preview-title {
+        -webkit-line-clamp: 3;
+        word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)